apps/busybox/deployers/test.yml

```diff
@@ -0,0 +1,210 @@
+              args: 
+                executable: /bin/bash
+              register: result
+              failed_when: "'scaled' not in result.stdout"
+
+            - name: Obtain number of healthy PVC
+              shell: >
+                kubectl get pvc -n {{ app_ns }} -l {{ app_label }}  -o custom-columns=:.status.phase | grep Bound | wc -l
+              args:
+                executable: /bin/bash
+              register: pvc
+
+            - name: Get the pv list after scale up the application replica
+              shell: >
+                kubectl get pvc -n {{ app_ns }} -l {{ app_label }} -o custom-columns=:.spec.volumeName --no-headers | grep -v none | wc -l
+              args:
+                executable: /bin/bash
+              register: pvlist
+
+            - name: Check if all the application replicas are running.
+              shell: kubectl get statefulset -n {{ app_ns }} --no-headers -l {{ app_label }} -o custom-columns=:..readyReplicas
+              args:
+                executable: /bin/bash
+              register: running_replicas
+              until: "running_replicas.stdout|int == pvlist.stdout|int == pvc.stdout|int"
+              delay: 60
+              retries: 15
+              
+            - name: Get the pv list after scale up the application replica
+              shell: >
+                kubectl get pvc -n {{ app_ns }} -l {{ app_label }} -o custom-columns=:.spec.volumeName --no-headers
+              args:
+                executable: /bin/bash
+              register: pv
+
+            - name: Obtain newly created pv list
+              set_fact:
+                 new_pv: "{{ (pv.stdout_lines | difference(pv_list.stdout_lines)) }}"
+
+            - name: Get the cstor storage pool list after scale up the application replica
+              shell: >
+                 kubectl get cvr -l openebs.io/replica-anti-affinity={{ app_lvalue }} -n {{ operator_ns }} -o jsonpath='{.items[*].metadata.labels.cstorpool\.openebs\.io/name}'
+              args:
+                executable: /bin/bash
+              register: new_csp
+
+            - set_fact:
+                 new_csp_list: "{{ new_csp.stdout.split(' ') }}"
+
+            - name: Verify the CVR's sceduled on different cstor storage pool
+              shell: >
+                kubectl get cvr -n {{ operator_ns }} -l openebs.io/replica-anti-affinity={{ app_lvalue }},cstorpool.openebs.io/name={{ item }} --no-headers | wc -l
+              args:
+                executable: /bin/bash
+              register: cvr_list
+              failed_when: cvr_list.stdout != '1'
+              with_items:
+                 - "{{ new_csp_list }}"
+
+          when: "'deprovision' not in action"
+
+        - block:
+
+            - name: Deprovisioning the Application
+              include_tasks: /common/utils/deprovision_statefulset.yml
+              vars:
+                app_deployer: "{{ application_deployment }}"
+
+          when: "'deprovision' is in action"
```

Comment:
@nsathyaseelan can you attempt deprovision? And place this directory under functional.

Reply:
@gprasath tried deprovision also
```
No config file found; using defaults
/etc/ansible/hosts did not meet host_list requirements, check plugin documentation if this is unexpected
/etc/ansible/hosts did not meet script requirements, check plugin documentation if this is unexpected
 [WARNING]: Found variable using reserved name: action

PLAY [localhost] ***************************************************************
2019-03-08T09:41:10.732043 (delta: 0.063837)         elapsed: 0.063837 ******** 
=============================================================================== 

TASK [Gathering Facts] *********************************************************
2019-03-08T09:41:10.748226 (delta: 0.016131)         elapsed: 0.08002 ********* 
ok: [127.0.0.1]

TASK [include_tasks] ***********************************************************
2019-03-08T09:41:20.723469 (delta: 9.975222)         elapsed: 10.055263 ******* 
included: /common/utils/create_testname.yml for 127.0.0.1

TASK [Record test instance/run ID] *********************************************
2019-03-08T09:41:20.868192 (delta: 0.144661)         elapsed: 10.199986 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Construct testname appended with runID] **********************************
2019-03-08T09:41:20.992132 (delta: 0.123839)         elapsed: 10.323926 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [include_tasks] ***********************************************************
2019-03-08T09:41:21.116391 (delta: 0.12416)         elapsed: 10.448185 ******** 
included: /common/utils/update_litmus_result_resource.yml for 127.0.0.1

TASK [Generate the litmus result CR to reflect SOT (Start of Test)] ************
2019-03-08T09:41:21.355985 (delta: 0.239493)         elapsed: 10.687779 ******* 
changed: [127.0.0.1] => {"changed": true, "checksum": "90d9ad54a895f3d10a73f17fa97d90a4b47eeaa5", "dest": "./litmus-result.yaml", "gid": 0, "group": "root", "md5sum": "239780ad5408fdda9046b3cd09041578", "mode": "0644", "owner": "root", "size": 421, "src": "/root/.ansible/tmp/ansible-tmp-1552038081.49-97117098894783/source", "state": "file", "uid": 0}

TASK [Analyze the cr yaml] *****************************************************
2019-03-08T09:41:22.450403 (delta: 1.094316)         elapsed: 11.782197 ******* 
changed: [127.0.0.1] => {"changed": true, "cmd": "cat litmus-result.yaml", "delta": "0:00:01.205557", "end": "2019-03-08 09:41:24.207806", "rc": 0, "start": "2019-03-08 09:41:23.002249", "stderr": "", "stderr_lines": [], "stdout": "---\napiVersion: litmus.io/v1alpha1\nkind: LitmusResult\nmetadata:\n\n  # name of the litmus testcase\n  name: replica-distribution \nspec:\n\n  # holds information on the testcase\n  testMetadata:\n    app:  \n    chaostype:  \n\n  # holds the state of testcase,  manually updated by json merge patch\n  # result is the useful value today, but anticipate phase use in future \n  testStatus: \n    phase: in-progress  \n    result: none ", "stdout_lines": ["---", "apiVersion: litmus.io/v1alpha1", "kind: LitmusResult", "metadata:", "", "  # name of the litmus testcase", "  name: replica-distribution ", "spec:", "", "  # holds information on the testcase", "  testMetadata:", "    app:  ", "    chaostype:  ", "", "  # holds the state of testcase,  manually updated by json merge patch", "  # result is the useful value today, but anticipate phase use in future ", "  testStatus: ", "    phase: in-progress  ", "    result: none "]}

TASK [Apply the litmus result CR] **********************************************
2019-03-08T09:41:24.265622 (delta: 1.815158)         elapsed: 13.597416 ******* 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl apply -f litmus-result.yaml", "delta": "0:00:02.239733", "end": "2019-03-08 09:41:26.780382", "failed_when_result": false, "rc": 0, "start": "2019-03-08 09:41:24.540649", "stderr": "", "stderr_lines": [], "stdout": "litmusresult.litmus.io/replica-distribution configured", "stdout_lines": ["litmusresult.litmus.io/replica-distribution configured"]}

TASK [Generate the litmus result CR to reflect EOT (End of Test)] **************
2019-03-08T09:41:26.841251 (delta: 2.575571)         elapsed: 16.173045 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Analyze the cr yaml] *****************************************************
2019-03-08T09:41:26.892799 (delta: 0.05149)         elapsed: 16.224593 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Apply the litmus result CR] **********************************************
2019-03-08T09:41:26.943232 (delta: 0.050376)         elapsed: 16.275026 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Get the storage class yaml] **********************************************
2019-03-08T09:41:26.999623 (delta: 0.056333)         elapsed: 16.331417 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Update the replica count storage class yaml] *****************************
2019-03-08T09:41:27.062320 (delta: 0.062633)         elapsed: 16.394114 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Update the storage class yaml] *******************************************
2019-03-08T09:41:27.133320 (delta: 0.070937)         elapsed: 16.465114 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Create the storage class with single replica] ****************************
2019-03-08T09:41:27.209991 (delta: 0.076605)         elapsed: 16.541785 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [include_tasks] ***********************************************************
2019-03-08T09:41:27.336814 (delta: 0.126718)         elapsed: 16.668608 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Replace the storage class in application deployment file] ****************
2019-03-08T09:41:27.474636 (delta: 0.137677)         elapsed: 16.80643 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Replace the replica anti affinity label value] ***************************
2019-03-08T09:41:27.596660 (delta: 0.121895)         elapsed: 16.928454 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [include_tasks] ***********************************************************
2019-03-08T09:41:27.717288 (delta: 0.120488)         elapsed: 17.049082 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Get the pv list] *********************************************************
2019-03-08T09:41:27.843676 (delta: 0.126281)         elapsed: 17.17547 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Obtain the pool claim name from storage class] ***************************
2019-03-08T09:41:27.967377 (delta: 0.123537)         elapsed: 17.299171 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Get the total number of sp] **********************************************
2019-03-08T09:41:28.083883 (delta: 0.1164)         elapsed: 17.415677 ********* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Obtain the cstor storage pool list from cvr] *****************************
2019-03-08T09:41:28.142610 (delta: 0.058666)         elapsed: 17.474404 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [set_fact] ****************************************************************
2019-03-08T09:41:28.199455 (delta: 0.056789)         elapsed: 17.531249 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Verify the CVR's sceduled on different sp] *******************************
2019-03-08T09:41:28.272641 (delta: 0.073131)         elapsed: 17.604435 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Obtain the application name] *********************************************
2019-03-08T09:41:28.362314 (delta: 0.089604)         elapsed: 17.694108 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Scaleup the application replica] *****************************************
2019-03-08T09:41:28.463377 (delta: 0.100977)         elapsed: 17.795171 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Obtain number of healthy PVC] ********************************************
2019-03-08T09:41:28.581505 (delta: 0.118026)         elapsed: 17.913299 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Get the pv list after scale up the application replica] ******************
2019-03-08T09:41:28.643344 (delta: 0.06178)         elapsed: 17.975138 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Check if all the application replicas are running.] **********************
2019-03-08T09:41:28.717192 (delta: 0.073792)         elapsed: 18.048986 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Get the pv list after scale up the application replica] ******************
2019-03-08T09:41:28.806921 (delta: 0.089623)         elapsed: 18.138715 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Obtain newly created pv list] ********************************************
2019-03-08T09:41:28.922288 (delta: 0.115288)         elapsed: 18.254082 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Get the cstor storage pool list after scale up the application replica] ***
2019-03-08T09:41:29.048609 (delta: 0.126216)         elapsed: 18.380403 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [set_fact] ****************************************************************
2019-03-08T09:41:29.128244 (delta: 0.079526)         elapsed: 18.460038 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Verify the CVR's sceduled on different cstor storage pool] ***************
2019-03-08T09:41:29.199000 (delta: 0.070699)         elapsed: 18.530794 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Deprovisioning the Application] ******************************************
2019-03-08T09:41:29.300541 (delta: 0.101448)         elapsed: 18.632335 ******* 
included: /common/utils/deprovision_statefulset.yml for 127.0.0.1

TASK [Check if the statefulset application exists.] ****************************
2019-03-08T09:41:29.591716 (delta: 0.291089)         elapsed: 18.92351 ******** 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get pods -n app-busybox-ns -l app=busybox-sts", "delta": "0:00:01.484020", "end": "2019-03-08 09:41:31.330349", "failed_when_result": false, "rc": 0, "start": "2019-03-08 09:41:29.846329", "stderr": "", "stderr_lines": [], "stdout": "NAME        READY   STATUS    RESTARTS   AGE\nbusybox-0   1/1     Running   0          5m\nbusybox-1   1/1     Running   0          5m", "stdout_lines": ["NAME        READY   STATUS    RESTARTS   AGE", "busybox-0   1/1     Running   0          5m", "busybox-1   1/1     Running   0          5m"]}

TASK [Obtaining PVCs related to the application.] ******************************
2019-03-08T09:41:31.417678 (delta: 1.825886)         elapsed: 20.749472 ******* 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get pvc -n app-busybox-ns -l app=busybox-sts --no-headers -o custom-columns=:.metadata.name", "delta": "0:00:01.239507", "end": "2019-03-08 09:41:33.058672", "rc": 0, "start": "2019-03-08 09:41:31.819165", "stderr": "", "stderr_lines": [], "stdout": "openebs-busybox-busybox-0\nopenebs-busybox-busybox-1", "stdout_lines": ["openebs-busybox-busybox-0", "openebs-busybox-busybox-1"]}

TASK [Obtaining the PV names.] *************************************************
2019-03-08T09:41:33.123264 (delta: 1.705473)         elapsed: 22.455058 ******* 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get pvc -l app=busybox-sts -n app-busybox-ns --no-headers -o custom-columns=:.spec.volumeName", "delta": "0:00:01.583657", "end": "2019-03-08 09:41:35.030565", "rc": 0, "start": "2019-03-08 09:41:33.446908", "stderr": "", "stderr_lines": [], "stdout": "pvc-8901388c-4185-11e9-ac71-0050569866b9\npvc-9cf997d1-4185-11e9-ac71-0050569866b9", "stdout_lines": ["pvc-8901388c-4185-11e9-ac71-0050569866b9", "pvc-9cf997d1-4185-11e9-ac71-0050569866b9"]}

TASK [Check for presence & value of cas type annotation] ***********************
2019-03-08T09:41:35.172419 (delta: 2.04909)         elapsed: 24.504213 ******** 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get pv pvc-8901388c-4185-11e9-ac71-0050569866b9 --no-headers -o jsonpath=\"{.metadata.annotations.openebs\\\\.io/cas-type}\"", "delta": "0:00:01.466162", "end": "2019-03-08 09:41:36.916639", "rc": 0, "start": "2019-03-08 09:41:35.450477", "stderr": "", "stderr_lines": [], "stdout": "cstor", "stdout_lines": ["cstor"]}

TASK [Record the storage engine name] ******************************************
2019-03-08T09:41:37.015177 (delta: 1.842654)         elapsed: 26.346971 ******* 
ok: [127.0.0.1] => {"ansible_facts": {"stg_engine": "cstor"}, "changed": false}

TASK [Replace the PVC name in application deployer spec.] **********************
2019-03-08T09:41:37.172306 (delta: 0.15704)         elapsed: 26.5041 ********** 
changed: [127.0.0.1] => {"changed": true, "msg": "2 replacements made"}

TASK [Replace the storageclass placeholder with provider] **********************
2019-03-08T09:41:37.791762 (delta: 0.619374)         elapsed: 27.123556 ******* 
changed: [127.0.0.1] => {"changed": true, "msg": "1 replacements made"}

TASK [Get the application replica values from env] *****************************
2019-03-08T09:41:38.160717 (delta: 0.368898)         elapsed: 27.492511 ******* 
ok: [127.0.0.1] => {"ansible_facts": {"app_rkey": "replicas", "app_rvalue": "2"}, "changed": false}

TASK [Replace the application label placeholder] *******************************
2019-03-08T09:41:38.352157 (delta: 0.191368)         elapsed: 27.683951 ******* 
changed: [127.0.0.1] => {"changed": true, "msg": "1 replacements made"}

TASK [Get the application label values from env] *******************************
2019-03-08T09:41:38.792694 (delta: 0.440435)         elapsed: 28.124488 ******* 
ok: [127.0.0.1] => {"ansible_facts": {"app_lkey": "app", "app_lvalue": "busybox-sts"}, "changed": false}

TASK [Replace the application label placeholder] *******************************
2019-03-08T09:41:38.907043 (delta: 0.114272)         elapsed: 28.238837 ******* 
changed: [127.0.0.1] => {"changed": true, "msg": "5 replacements made"}

TASK [Delete the application and its related service.] *************************
2019-03-08T09:41:39.249825 (delta: 0.342713)         elapsed: 28.581619 ******* 
changed: [127.0.0.1] => {"attempts": 1, "changed": true, "cmd": "kubectl delete -f busybox_statefulset.yml -n app-busybox-ns", "delta": "0:00:01.840997", "end": "2019-03-08 09:41:41.398853", "rc": 0, "start": "2019-03-08 09:41:39.557856", "stderr": "", "stderr_lines": [], "stdout": "service \"busybox\" deleted\nstatefulset.apps \"busybox\" deleted", "stdout_lines": ["service \"busybox\" deleted", "statefulset.apps \"busybox\" deleted"]}

TASK [Deleting the PVC] ********************************************************
2019-03-08T09:41:41.481955 (delta: 2.232074)         elapsed: 30.813749 ******* 
changed: [127.0.0.1] => (item=openebs-busybox-busybox-0) => {"changed": true, "cmd": "kubectl delete pvc openebs-busybox-busybox-0 -n app-busybox-ns", "delta": "0:00:31.283147", "end": "2019-03-08 09:42:13.101824", "item": "openebs-busybox-busybox-0", "rc": 0, "start": "2019-03-08 09:41:41.818677", "stderr": "", "stderr_lines": [], "stdout": "persistentvolumeclaim \"openebs-busybox-busybox-0\" deleted", "stdout_lines": ["persistentvolumeclaim \"openebs-busybox-busybox-0\" deleted"]}
changed: [127.0.0.1] => (item=openebs-busybox-busybox-1) => {"changed": true, "cmd": "kubectl delete pvc openebs-busybox-busybox-1 -n app-busybox-ns", "delta": "0:00:01.425260", "end": "2019-03-08 09:42:14.736186", "item": "openebs-busybox-busybox-1", "rc": 0, "start": "2019-03-08 09:42:13.310926", "stderr": "", "stderr_lines": [], "stdout": "persistentvolumeclaim \"openebs-busybox-busybox-1\" deleted", "stdout_lines": ["persistentvolumeclaim \"openebs-busybox-busybox-1\" deleted"]}

TASK [Check if the PVCs are deleted] *******************************************
2019-03-08T09:42:14.795433 (delta: 33.313423)         elapsed: 64.127227 ****** 
changed: [127.0.0.1] => {"attempts": 1, "changed": true, "cmd": "kubectl get pvc -n app-busybox-ns", "delta": "0:00:01.334518", "end": "2019-03-08 09:42:16.395369", "rc": 0, "start": "2019-03-08 09:42:15.060851", "stderr": "No resources found.", "stderr_lines": ["No resources found."], "stdout": "", "stdout_lines": []}

TASK [Check if the ctrl-pod is deleted.] ***************************************
2019-03-08T09:42:16.458311 (delta: 1.662822)         elapsed: 65.790105 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Check if the  replica-pod is deleted.] ***********************************
2019-03-08T09:42:16.523878 (delta: 0.065512)         elapsed: 65.855672 ******* 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Check if the target pod is deleted.] *************************************
2019-03-08T09:42:16.589334 (delta: 0.0654)         elapsed: 65.921128 ********* 
FAILED - RETRYING: Check if the target pod is deleted. (20 retries left).
FAILED - RETRYING: Check if the target pod is deleted. (19 retries left).
FAILED - RETRYING: Check if the target pod is deleted. (18 retries left).
FAILED - RETRYING: Check if the target pod is deleted. (17 retries left).
FAILED - RETRYING: Check if the target pod is deleted. (16 retries left).
changed: [127.0.0.1] => (item=pvc-8901388c-4185-11e9-ac71-0050569866b9) => {"attempts": 6, "changed": true, "cmd": "kubectl get pods -l openebs.io/persistent-volume=pvc-8901388c-4185-11e9-ac71-0050569866b9 -n openebs --no-headers", "delta": "0:00:02.444358", "end": "2019-03-08 09:42:52.677536", "item": "pvc-8901388c-4185-11e9-ac71-0050569866b9", "rc": 0, "start": "2019-03-08 09:42:50.233178", "stderr": "No resources found.", "stderr_lines": ["No resources found."], "stdout": "", "stdout_lines": []}
changed: [127.0.0.1] => (item=pvc-9cf997d1-4185-11e9-ac71-0050569866b9) => {"attempts": 1, "changed": true, "cmd": "kubectl get pods -l openebs.io/persistent-volume=pvc-9cf997d1-4185-11e9-ac71-0050569866b9 -n openebs --no-headers", "delta": "0:00:01.262187", "end": "2019-03-08 09:42:54.356497", "item": "pvc-9cf997d1-4185-11e9-ac71-0050569866b9", "rc": 0, "start": "2019-03-08 09:42:53.094310", "stderr": "No resources found.", "stderr_lines": ["No resources found."], "stdout": "", "stdout_lines": []}

TASK [Check status of the pool deployments from cvr] ***************************
2019-03-08T09:42:54.458366 (delta: 37.868975)         elapsed: 103.79016 ****** 
changed: [127.0.0.1] => (item=pvc-8901388c-4185-11e9-ac71-0050569866b9) => {"attempts": 1, "changed": true, "cmd": "kubectl get cvr -n openebs -l openebs.io/persistent-volume=pvc-8901388c-4185-11e9-ac71-0050569866b9 --no-headers", "delta": "0:00:01.369476", "end": "2019-03-08 09:42:56.109774", "item": "pvc-8901388c-4185-11e9-ac71-0050569866b9", "rc": 0, "start": "2019-03-08 09:42:54.740298", "stderr": "No resources found.", "stderr_lines": ["No resources found."], "stdout": "", "stdout_lines": []}
changed: [127.0.0.1] => (item=pvc-9cf997d1-4185-11e9-ac71-0050569866b9) => {"attempts": 1, "changed": true, "cmd": "kubectl get cvr -n openebs -l openebs.io/persistent-volume=pvc-9cf997d1-4185-11e9-ac71-0050569866b9 --no-headers", "delta": "0:00:01.274892", "end": "2019-03-08 09:42:57.759675", "item": "pvc-9cf997d1-4185-11e9-ac71-0050569866b9", "rc": 0, "start": "2019-03-08 09:42:56.484783", "stderr": "No resources found.", "stderr_lines": ["No resources found."], "stdout": "", "stdout_lines": []}

TASK [Delete the namespace.] ***************************************************
2019-03-08T09:42:57.829795 (delta: 3.371346)         elapsed: 107.161589 ****** 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl delete ns app-busybox-ns", "delta": "0:00:06.769291", "end": "2019-03-08 09:43:04.934641", "rc": 0, "start": "2019-03-08 09:42:58.165350", "stderr": "", "stderr_lines": [], "stdout": "namespace \"app-busybox-ns\" deleted", "stdout_lines": ["namespace \"app-busybox-ns\" deleted"]}

TASK [set_fact] ****************************************************************
2019-03-08T09:43:04.987352 (delta: 7.157469)         elapsed: 114.319146 ****** 
ok: [127.0.0.1] => {"ansible_facts": {"flag": "Pass"}, "changed": false}

TASK [include_tasks] ***********************************************************
2019-03-08T09:43:05.065679 (delta: 0.078273)         elapsed: 114.397473 ****** 
included: /common/utils/update_litmus_result_resource.yml for 127.0.0.1

TASK [Generate the litmus result CR to reflect SOT (Start of Test)] ************
2019-03-08T09:43:05.218128 (delta: 0.152371)         elapsed: 114.549922 ****** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Analyze the cr yaml] *****************************************************
2019-03-08T09:43:05.285360 (delta: 0.067185)         elapsed: 114.617154 ****** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Apply the litmus result CR] **********************************************
2019-03-08T09:43:05.341582 (delta: 0.056153)         elapsed: 114.673376 ****** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Generate the litmus result CR to reflect EOT (End of Test)] **************
2019-03-08T09:43:05.452725 (delta: 0.111047)         elapsed: 114.784519 ****** 
changed: [127.0.0.1] => {"changed": true, "checksum": "fd0605c6bff14796e031242127572310e68dc178", "dest": "./litmus-result.yaml", "gid": 0, "group": "root", "md5sum": "db8a69568de9d7d3e74f4ed730b643e0", "mode": "0644", "owner": "root", "size": 419, "src": "/root/.ansible/tmp/ansible-tmp-1552038185.58-8904623693264/source", "state": "file", "uid": 0}

TASK [Analyze the cr yaml] *****************************************************
2019-03-08T09:43:08.468332 (delta: 3.015464)         elapsed: 117.800126 ****** 
changed: [127.0.0.1] => {"changed": true, "cmd": "cat litmus-result.yaml", "delta": "0:00:01.303629", "end": "2019-03-08 09:43:10.103210", "rc": 0, "start": "2019-03-08 09:43:08.799581", "stderr": "", "stderr_lines": [], "stdout": "---\napiVersion: litmus.io/v1alpha1\nkind: LitmusResult\nmetadata:\n\n  # name of the litmus testcase\n  name: replica-distribution \nspec:\n\n  # holds information on the testcase\n  testMetadata:\n    app:  \n    chaostype:  \n\n  # holds the state of testcase,  manually updated by json merge patch\n  # result is the useful value today, but anticipate phase use in future \n  testStatus: \n    phase: completed  \n    result: Pass ", "stdout_lines": ["---", "apiVersion: litmus.io/v1alpha1", "kind: LitmusResult", "metadata:", "", "  # name of the litmus testcase", "  name: replica-distribution ", "spec:", "", "  # holds information on the testcase", "  testMetadata:", "    app:  ", "    chaostype:  ", "", "  # holds the state of testcase,  manually updated by json merge patch", "  # result is the useful value today, but anticipate phase use in future ", "  testStatus: ", "    phase: completed  ", "    result: Pass "]}

TASK [Apply the litmus result CR] **********************************************
2019-03-08T09:43:10.163041 (delta: 1.694626)         elapsed: 119.494835 ****** 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl apply -f litmus-result.yaml", "delta": "0:00:01.573760", "end": "2019-03-08 09:43:12.071470", "failed_when_result": false, "rc": 0, "start": "2019-03-08 09:43:10.497710", "stderr": "", "stderr_lines": [], "stdout": "litmusresult.litmus.io/replica-distribution configured", "stdout_lines": ["litmusresult.litmus.io/replica-distribution configured"]}

PLAY RECAP *********************************************************************
127.0.0.1                  : ok=29   changed=20   unreachable=0    failed=0   

2019-03-08T09:43:12.146417 (delta: 1.98331)         elapsed: 121.478211 ******* 
=============================================================================== 
```

Reply:
Thanks @nsathyaseelan